war/src/main/less/base/style.less

```diff
@@ -573,7 +573,7 @@ pre.console {
   }
 
   p:last-of-type {
-    margin-bottom: 0;
+    margin-bottom: 10px;
```

Comment:
What exactly is this change for? It adds more space to the last row of all items for no obvious reason to me.

Reply:
I think it look more tidy, so I add some space in the bottom, if this is redundant I have reverted, please take a look.